### ISSUE TITLE
Chart title does not obey responsive layout

### ISSUE THREAD

**Issue by aadibajpai** [plotly/plotly.js#5109]
Hi, I was following this https://plotly.com/javascript/responsive-fluid-layout/ and it made my chart responsive but the title does not wrap and gets truncated on mobile. 

![image](https://user-images.githubusercontent.com/27063113/91669180-ac4e6d80-eb30-11ea-8f6f-ad0041518c05.png)

Link to code: https://gist.github.com/aadibajpai/b0c11721056325d4892cf926b950cdb1#file-snippet-html

and link to production: https://vanderbilthustler.com/category/coronavirus/

**Comment by nicolaskruchten** [comment]
Yes, in general Plotly.js doesn't have any kind of automatic text-wrapping yet, unfortunately.

**Comment by Mukulikaa** [comment]
Is there a workaround to this?
